lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -214,11 +215,13 @@ class XMLTemplateAnalyzer {
 		const localName = node.$ns.local;
 
 		const oldConditional = this.conditional;
+		const oldTemplateTag = this.templateTag;
 
 		if ( namespace === TEMPLATING_NAMESPACE ) {
 			if ( TEMPLATING_CONDITONAL_TAGS.test(localName) ) {
```

Comment:
Sorry, I missed this if-clause. Therefore we really need to have two separate flags I think.

Reply:
np :)